docs/en/ingest-management/fleet/upgrade-elastic-agent.asciidoc

```diff
@@ -1,15 +1,25 @@
 [[upgrade-elastic-agent]]
 = Upgrade {fleet}-managed {agent}s
 
-experimental[]
-
 ++++
 <titleabbrev>Upgrade {agent}s</titleabbrev>
 ++++
 
+TIP: Want to upgrade a standalone agent instead? See <<upgrade-standalone>>.
+
 With {fleet} upgrade capabilities, you can view and select agents that are out
 of date, and trigger selected agents to download, install, and run the new
-version.
+version. You can trigger upgrades to happen immediately, or specify a
+maintenance window during which the upgrades will occur. If your {stack}
+subscription level supports it, you can schedule upgrades to occur at a specific
+date and time.
+
+//REVIEWERS: From the design doc, it seemed like scheduled upgrades and
+//maintenance windows are only available to some subscription levels. But looking
```

Comment:
Maintenance windows are available at all subscription level (only the schedule part is licence restricted)